{
  try {
    log.logMessage(msg,exception,true);
  }
 catch (  Throwable t) {
  }
  return exception;
}
